apps/settings/js/panels/languages/languages.js

```diff
@@ -1,7 +1,10 @@
+/* global MozActivity */
+
```

Comment:
we can use `window.MozActivity` below, in this way, we don't need to put this extra line here.